### PR TITLE
added fetchData.url

### PR DESCRIPTION
**Pull request by lawinslow**
@jiwalker-usgs @aappling-usgs Want to take a look? Pairs with the climate-fish-hab PR.

### CODE REVIEW

**Comment by aappling-usgs** [comment]
looks great to me. i'll wait a sec to give @jiwalker-usgs time to look, but i think this is ready for merging.

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Package: vizlab
 Type: Package
 Title: Utilities for building online data visualizations
-Version: 0.1.0
+Version: 0.1.1
```
Comment (jiwalker-usgs) [inline DESCRIPTION]: I wouldn't worry too much about incrementing the version yet, but :+1:

**Comment by jiwalker-usgs** [comment]
:+1: what do you think of this pattern in general?

File: DESCRIPTION (hunk 1 above)
Comment (lawinslow) [inline DESCRIPTION]: I like to stay in the habit.

**Comment by lawinslow** [comment]
Looks good to me. Should the PR be failing?

**Comment by jiwalker-usgs** [comment]
Still need to fix the build

**Comment by lawinslow** [comment]
Ah, got it.

File: R/fetchData.R [hunk 2]
```diff
@@ -62,3 +62,21 @@ fetchData.sciencebase <- function(viz.id, data.info) {
...
   invisible()
 }
+
+#' \code{fetchData.url} downloads a file from the specified URL
+#' 
+#' @param viz.id the identifier for this data item in viz.yaml
+#' @param data.info content information for this viz.id from the viz.yaml
+#' @rdname fetchData
+#' @export
+fetchData.url <- function(viz.id, data.info){
```
Comment (jread-usgs) [inline R/fetchData.R]: `@importFrom httr GET` ?

Comment (lawinslow) [inline R/fetchData.R]: What's the strategy with this package. Sbtools is just suggests so I was leaving this suggests as well.

Comment (jiwalker-usgs) [inline R/fetchData.R]: Yeah, I started using suggests to keep the actual dependencies to a minimum.  I'm not sure if that makes things easier or harder.

Comment (lawinslow) [inline R/fetchData.R]: I say bounce them both to imports. sbtools should be on CRAN soon. I always like it when I can keep a CRAN-only imports list.

**Pull request merged**